Source/WebCore/Modules/screen-wake-lock/WakeLockManager.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "Document.h"
```

Comment:
We could have moved the functions out of line to avoid extra includes in the header and slowing down the build.

Reply:
Fixed.